context-extraction/src/main/java/de/atb/context/services/ContextRepositoryService.java

```diff
@@ -148,12 +135,10 @@ public final synchronized void persistContext(final ContextContainerWrapper wrap
         }
     }
 
-    /*
+    /**
 	 * (non-Javadoc)
-	 * 
-	 * @see
-	 * de.atb.context.persistence.IContextRepository#getContext(de.atb.
-	 * context.common.util.BusinessCase, java.lang.String)
+	 *
+	 * @see IContextRepositoryService#getContext(de.atb.context.common.util.BusinessCase, java.lang.String)
```

Comment:
shouldn't there be : `* @see IContextRepositoryService#getContext(de.atb.context.common.util.ApplicationScenario, java.lang.String)` ?

Reply:
changed